sdk/eventhub/azure-eventhubs/conftest.py

```diff
@@ -19,14 +19,23 @@
     collect_ignore.append("features")
     collect_ignore.append("examples/async_examples")
 else:
-    from azure.eventprocessorhost import EventProcessorHost
-    from azure.eventprocessorhost import EventHubPartitionPump
-    from azure.eventprocessorhost import AzureStorageCheckpointLeaseManager
-    from azure.eventprocessorhost import AzureBlobLease
-    from azure.eventprocessorhost import EventHubConfig
-    from azure.eventprocessorhost.lease import Lease
-    from azure.eventprocessorhost.partition_pump import PartitionPump
-    from azure.eventprocessorhost.partition_manager import PartitionManager
+    try:
+        from azure.eventprocessorhost import EventProcessorHost
+        from azure.eventprocessorhost import EventHubPartitionPump
```

Comment:
I noticed the event hub checks were running, is this an intentional change for this PR?

Reply:
Yeah - because eventhubs had a dependency on a previous version of storage I added this check. We're making modifications to our CI so that EH tests can run with the correct storage version independently.